### ISSUE TITLE
Broken symlink in docs

### ISSUE THREAD

**Issue by rybakit** [amphp/byte-stream#48]
Installing this library (as a dependency of `amphp/parallel`) through composer results in a broken symlink in the docs directory:

```bash
$ ls -la vendor/amphp/byte-stream/docs
lrwxrwxrwx. 1 gen gen   13 Dec 27 15:28 asset -> .shared/asset
...
drwxrwxr-x. 2 gen gen   40 Oct 22 21:37 .shared
```
```bash
$ ls -la vendor/amphp/byte-stream/docs/.shared
total 0
drwxrwxr-x. 2 gen gen  40 Oct 22 21:37 .
drwxrwxr-x. 3 gen gen 300 Oct 22 21:37 ..
```

This broken link causes Gitlab CI to generate a warning while buliding a project:

```bash
No URL provided, cache will be not downloaded from shared cache server. Instead a local version of cache will be extracted. 
WARNING: .../vendor/amphp/byte-stream/docs/asset: chmod .../vendor/amphp/byte-stream/docs/asset: no such file or directory (suppressing repeats) 
```
(see a related [Gitlab issue](https://gitlab.com/gitlab-org/gitlab-ce/issues/19770)).

**Comment by kelunik** [comment]
Fixed via 6bbfcb6f47e92577e739586ba0c87e867be70a23.

**Comment by kelunik** [comment]
Released `v1.5.1`.
